applications/GeoMechanicsApplication/tests/cpp_tests/test_U_Pw_normal_lysmer_absorbing_condition.cpp

```diff
@@ -0,0 +1,264 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Aron Noordam
+//
+
+#include <string>
+
+// Project includes
+#include "custom_elements/plane_strain_stress_state.h"
+#include "custom_elements/U_Pw_small_strain_element.hpp"
+
+#include "geo_mechanics_fast_suite.h"
+
+namespace Kratos::Testing {
+
+    class MockElement : public UPwSmallStrainElement<2, 4>
+    {
+    public:
+
+        MockElement() : UPwSmallStrainElement<2, 4>() {}
+
+        MockElement(IndexType NewId, GeometryType::Pointer pGeometry, PropertiesType::Pointer pProperties)
+            : UPwSmallStrainElement(NewId, pGeometry, pProperties, std::make_unique<PlaneStrainStressState>())
+        {
+        }
+
+        Element::Pointer Create(IndexType NewId, GeometryType::Pointer pGeometry, PropertiesType::Pointer pProperties) const override
+        {
+            return Kratos::make_intrusive<MockElement>(NewId, pGeometry, pProperties);
+        }
```

Comment:
A bit further in the test, I have a suggestion to not use the kratos registry for the mock element. If that works for you, you can probably remove this function (don't think that's needed anymore)